CefSharp.BrowserSubprocess.Core/CefSharp.BrowserSubprocess.Core.vcxproj

```diff
@@ -73,25 +73,29 @@
     <IgnoreImportLibrary>true</IgnoreImportLibrary>
     <OutDir>bin\$(Platform)\$(Configuration)\</OutDir>
     <IntDir>obj\$(Platform)\$(Configuration)\</IntDir>
+    <IncludePath>$(SolutionDir)packages\$(CefSdkVer)\CEF;$(IncludePath)</IncludePath>
```

Comment:
Does using `SourcePath` work for you? Description given in `VS` being `Path to use when searching for source files to use for Intellisense`

```
<SourcePath>$(SolutionDir)packages\$(CefSdkVer)\CEF;$(SourcePath)</SourcePath>
```

Reply:
Yes, same effect.
Should probably switch over to that one then as that's what it's made for